src/Components/Admin/Rooms/roomModal.js

```diff
@@ -261,7 +261,11 @@ export default function RoomModal({
     };
 
     const handleModalCloseClick = () => {
-        if (isShareAndPublish) {
+        if (
```

Comment:
Should we have a toast message or something to show that the changes were saved? 
I guess we currently don't do that right now, so not a biggie.

Reply:
Definitely a nice to have throughout the app in the future. If we have time, I'll def do this!